src/main/pages/workspace-admin/recipes.adoc

```diff
@@ -100,34 +143,28 @@ networks:
   driver: bridge
 ----
 
-Che does not yet currently support compose networks. We do, however, support the use of aliases through `links` command.
-
 *Hostname*
 
-Hostname is not supported and the machine’s name will be used for the hostname. User can use `links` aliases syntax to add additional hostname(s) to a machine.
+The hostname is not supported.  The machine’s name is used for the hostname. You can use `links` aliases syntax to add additional hostnames to a machine.
 
-*Ports*
+*Binding ports*
 
-Binding ports to the host system is not supported. This is done to ensure that each container does not used already assigned host ports because many developers could be running the same compose file on the same Che server host at the same time. Users can add link:servers.html[servers] to machines though.
+Binding ports to the host system is not supported.  You must ensure that each container does not use already assigned host ports.  Many developers can be running the same Compose file on the same Che server host at the same time. You can add link:servers.html[servers] to the machines.
 
-*Privileged*
+*Environment file*
 
-To secure the underlying host system `privileged` compose command is not supported.
+The `env_file` compose command is not supported. Environment variables can be manually entered in the Compose file or machine configuration. See link:env-variables.html[Environment variables] for more information.
 
-The Che server can be configured to give all containers privileged access by setting the environment variable `pass:[CHE_PROPERTY_machine_docker_privilege__mode=true]`. However, this makes the host system vulnerable and gives all containers access to the host system.
+[id="kubernetes-yaml-limitations-and-restrictions"]
+== Kubernetes YAML limitations and restrictions
 
-*Environment File* The `env_file` compose command is not supported. Environment variables can be manually entered in compose file or machine configuration. See: link:env-variables.html[Environment variables].
+When a workspace is starting, Che creates a https://kubernetes.io/docs/concepts/workloads/pods/pod/[Kubernetes pod]. The following are limitatons and restrictions:
 
-[id="kubernetes-yaml"]
-== Kubernetes YAML
+1.  Che allows user to create only pods and services.
+2.  If you have an exception from Kubernetes, other object kinds such as PVCs and routes are ignored or a workspace fails to starts.
```

Comment:
This changes orig. meaning and makes no sense.

Originally, it said that when you're starting a workspace, and a pod is created, other (other then pods) objects are ignored (if specified). Sometimes, apparently, the workspace fails to start if other objects are specified, and K8s throws an exception.

There's no need to change the orig. sentence. If you want to improve it, inquire with an SME why the workspace creation only fails sometimes.